conftest.py

```diff
@@ -233,6 +236,32 @@ def pytest_sessionstart(session: Session) -> None:
         path=must_gather_dict["must_gather_base_directory"],
         ignore_errors=True,
     )
+    updated_global_config(admin_client=get_client())
```

Comment:
this should be skipped `--collect-only` or `--setup-plan` are passed. Otherwise you would see failures in tox.